lua/nordic/groups/native.lua

```diff
@@ -66,7 +66,7 @@ function M.get_groups()
     G.htmlH1 = { fg = C.yellow.base, bold = true }
     G.htmlH2 = { fg = C.orange.base }
     G.Link = { fg = C.blue1, underline = true }
-    G.CodeBlock = { bg = C.black1, fg = C.fg }
+    G.CodeBlock = { bg = C.bg_float, fg = C.fg }
```

Comment:
Code blocks are not floating?

Reply:
But their background needs to change with swap backgrounds. I can't just switch the value of black1 and gray0 otherwise its just a toggle. 

I needed a value to store the current background colors, that cold be swapped, bg & bg_float seemed like the right fit seeing as they are already doing this for most of the highlights..

Functionally the code is the exact same but if you would like I can make another value in `extend_palette` for them.

Reply:
Wait you right about these I need another color for them because, bg_float is not functionally the same... its transparent when the background is. I will fix these, thanks for pointing them out.